README.md

```diff
@@ -186,10 +233,16 @@ const p = div.getElementsByTagName('p')[0]
 p.textContent = "Goodbye!"
 ```
 
-Obviously, this way of accessing that text isn't fully generic, but it does a good job of demonstrating the basic tools available to us for finding and manipulating HTML elements.
+Obviously, this way of accessing that text isn't fully generic, but it does a
```

Comment:
i'd avoid generic here...

"it's inefficient" and won't work on all pages (~generic~)